src/api/app/assets/javascripts/webui/application.js

```diff
@@ -207,7 +207,7 @@ function change_role(obj) {
         complete: function () {
             $('#' + type + '_spinner').hide();
             $('#' + type + '_table input').animate({opacity: 1}, 200);
-            $('#' + type + '_table input').removeAttr('disabled');
+            $('#' + type + '_table input').prop('disabled', false);
```

Comment:
What do you mean @coolo? that the `prop` method doesn't work in jquery 2.1? I have tried this at build.opensuse.org in the javascript console and it works...